libbeat/version/version.go

```diff
@@ -18,4 +18,4 @@
 // Code generated by dev-tools/set_version
 package version
 
-const defaultBeatVersion = "9.0.0"
+const defaultBeatVersion = "8.16.0"
```

Comment:
are you sure that we need to change this to 8.16.0 here?! I have the impression that main needs to be 9.0.0 and 8.x at 8.16.0

Reply:
woops, wrong file 
will remove